certified-connectors/Converter by Power2Apps/apiDefinition.swagger.json

```diff
@@ -5,50 +5,61 @@
     "description": "Enables conversion of different data formats from one to another.",
     "contact": {
       "name": "Power2Apps Support",
-      "url": "https://www.power2apps.de/kontakt",
-      "email": "kontakt@power2apps.de"
+      "url": "https://www.power2apps.com/en/contact",
+      "email": "getintouch@power2apps.com"
     },
-    "version": "1.0.0"
+    "version": "1.0.7"
   },
   "host": "webapiconverterservice20211014190508.azurewebsites.net",
   "schemes": [
     "https"
   ],
-  "basePath": "/Converter",
   "paths": {
```

Comment:
Basepath has been changed, kindly confirm that that existing flows will not be broken, please refer the below link. 

https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-updates#notify-your-customer-of-breaking-changes